elixir/apps/web/lib/web/live/resources/show.ex

```diff
@@ -116,25 +116,19 @@ defmodule Web.Resources.Show do
               <%= @resource.address %>
             </:value>
           </.vertical_table_row>
-          <.vertical_table_row>
+          <.vertical_table_row :if={@resource.address_description}>
```

Comment:
We don't want to show the row at all if there is no address description?  Just wondering if that might throw some people off if they have some resources with descriptions and others that don't.

Reply:
Yeah I don't think this is consistent with other fields. Updated.